charts/seed-bootstrap/charts/vpa/templates/recommender-deployment.yaml

```diff
@@ -30,6 +30,13 @@ spec:
       containers:
       - name: recommender
         image: {{ index .Values.global.images "vpa-recommender" }}
+        command:
+        - ./recommender
+        args:
+        - --v=4
+        - --stderrthreshold=info
```

Comment:
Doesn't this spam logs a lot? For other K8S components, the log level is 2

Reply:
`v=4` was used in the Dockerfiles. I changed it to `v=2`.